.jenkins_quicktest

```diff
@@ -1,34 +1,38 @@
 #!/bin/bash
 set -e
 set -o pipefail
+export IBAMR_DIR=$PWD
+export IBAMR_ARCH=build
+export PETSC_DIR=$IBAMR_DIR/$IBAMR_ARCH/PETSC
+export PETSC_ARCH=linux-debug
 
-# export the following environment variables
-# before executing this script in the jenkins
-# execute shell
+build_petsc() {
+    mkdir -p $PETSC_DIR && git clone -b knepley/ibamr https://bitbucket.org/deleeke/petsc $PETSC_DIR
+    cd $PETSC_DIR
+    ./configure --download-openmpi --download-fblaslapack \
+        --CC=$C_COMPILER --CXX=$CXX_COMPILER --FC=$FORTRAN_COMPILER \
+        --with-clanguage=c++ --with-debugging=$DEBUG_OPTION
+    make
```

Comment:
If you want jenkins to work for this build these environment variables must be set in the build script before Jenkins executes this script.